app/build.gradle

```diff
@@ -39,7 +41,9 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.2.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.2'
+    implementation 'com.github.bumptech.glide:glide:4.11.0'
```

Comment:
I did consider using coil, as I see some people moving towards it, but I didn't want to use it just because other people are using without properly researching it. so I did stick with what I'm used to.